src/components/ExperienceContainer/ExperienceContainer.jsx

```diff
@@ -0,0 +1,169 @@
+import { useState, useEffect } from "react";
+import "./ExperienceContainer.css";
+
+export const ExperienceContainer = () => {
+  const [experienceList, setExperienceList] = useState([]);
+
+  useEffect(() => {
+    setExperienceList(mockExperience);
+  }, []);
+
+  // Saves the experience card by closing the form
+  const saveEditForm = (experience) => () => {
+    // Find the experience card whose id matches the experience id
+    const experienceCard = document.getElementById(
+      `experience${experience.id}`
+    );
+
+    // Get the values from the form
+    const company = document.getElementById("company").value;
+    const title = document.getElementById("title").value;
+    const startDate = document.getElementById("startDate").value;
+    const endDate = document.getElementById("endDate").value;
+    const description = document.getElementById("description").value;
+    const logo = document.getElementById("logo").value;
+
+    // Delete the content of the experience card
+    experienceCard.innerHTML = "";
+
+    // Append new content
+    const html = `<div class="content">
+                    <h3>
+                      ${company} - ${title}
```

Comment:
nit: setting inner html usually makes it easy for the XSS attacks, but it's not really part of this issue, so please use your best judgement about this. If you think you can fix it, go for it, if not, maybe we can track it as a separate issue and fix it for that.

Reply:
Would it be okay if I used an HTML sanitization library to parse the HTML (while keeping the innerHtml call)? It seems it's a [valid solution](https://coderpad.io/blog/development/javascript-innerhtml/):